components/gemini/browser/gemini_service_browsertest.cc

```diff
@@ -0,0 +1,475 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
if the tests have to use chrome/* then they should probably go in brave/browser/gemini